### PR TITLE
To Do CRUD (create, Update and remove)

### PR DESCRIPTION
**Pull request by christianbiring1**
In this project, I had to implement the CRUD (create, update, delete) methods. All the elements of the user interface will be fully functional and the application will be completed.

### Deployed page can be checked [here](https://christianbiring1.github.io/To-Do_list/)

### CODE REVIEW

File: src/modules/delete.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    const container = document.querySelector('.container');
+    btn.addEventListener('click', (e) => {
+      const item = e.target.parentNode.parentNode;
+      container.removeChild(item);
+      const deletedTask = e.target.previousElementSibling.textContent;
+      tasks = tasks.filter((task) => task.description !== deletedTask);
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+    });
+  });
+}
```
Comment (mikethreels) [inline src/modules/delete.js:17]: - [x] removing a task gives an error in the console and it removes the entire local storage as you can see in below print screen.
Screenshot 2022-04-29 203426
- [x] keep in mind that Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Christian, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] A function for editing task descriptions should be implemented, however, it is missing in this update.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/delete.js (hunk 1 above)
Comment (christianbiring1) [reply src/modules/delete.js:17]: when I open the app locally I'm not getting that `error` but on the deployed one is where I'm getting it. I don't really know what I can do to solve that. could you mind helping @mikethreels? 

The app is working fine I think, the user can add, and remove. tasks are saved in the local storage.

Comment (christianbiring1) [reply src/modules/delete.js:17]: The `error` of the local storage to be deleted I just fix it now. 

> About that error that is thrown in the console I've been looking all over the internet I can't find a way out. And you may notice in the dynamically rendered element, the element I'm removing is actually a child od the container buy I don't understand how I'm getting that error...

**Review by vic778 — APPROVED** [review]
Hi @christianbiring1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/delete.js (hunk 1 above)
Comment (mikethreels) [reply src/modules/delete.js:17]: Hi Christian,
Sorry for the late response, at the time you wrote to me I wasn't able to respond, great job fixing it 👏

**Pull request merged**